chainer_chemistry/links/readout/gin_readout.py

```diff
@@ -0,0 +1,55 @@
+import chainer
+from chainer import functions
+
+from chainer_chemistry.links.connection.graph_linear import GraphLinear
+
+
+class GINReadout(chainer.Chain):
```

Comment:
Since it is copied from GGNNReadout, we might remove this module and simply use GGNNReadout.